llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -1505,7 +1505,10 @@ class LoopVectorizationCostModel {
   /// Returns true if epilogue vectorization is considered profitable, and
   /// false otherwise.
   /// \p VF is the vectorization factor chosen for the original loop.
-  bool isEpilogueVectorizationProfitable(const ElementCount VF) const;
+  /// \p Multiplier is an aditional scaling factor applied to VF before
+  /// comparing to EpilogueVectorizationMinVF.
+  bool isEpilogueVectorizationProfitable(const ElementCount VF,
```

Comment:
`Multiplier` is the UF/IC, right? Might be clearer to call it that to avoid confusion

Reply:
Only in the non-SVE case, for SVE it's `getVScaleForTuning(OrigLoop, TTI).value_or(1)`. Which this wouldn't be changing just pulling it out as an argument and passing `IC` for non-SVE.